exonum-java-binding/cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/transactions/TransferTxIntegrationTest.java

```diff
@@ -65,37 +60,6 @@
   private static final KeyPair FROM_KEY_PAIR = PredefinedOwnerKeys.FIRST_OWNER_KEY_PAIR;
   private static final KeyPair TO_KEY_PAIR = PredefinedOwnerKeys.SECOND_OWNER_KEY_PAIR;
 
-  @Test
-  void from() {
-    long seed = 1;
-    long sum = 50L;
-    PublicKey recipientKey = TO_KEY_PAIR.getPublicKey();
-
-    byte[] arguments = newTransferTxPayload(seed, recipientKey, sum);
-
-    TransferTx tx = TransferTx.from(arguments);
-
-    assertThat(tx).isEqualTo(new TransferTx(seed, recipientKey, sum));
-  }
-
-  @ParameterizedTest
-  @ValueSource(longs = {
-      Long.MIN_VALUE,
-      -100,
-      -1,
-      0
-  })
-  void fromRawTransactionRejectsNonPositiveBalance(long transferAmount) {
```

Comment:
As the condition was moved from the constructor to execute, I'd keep the test as `executeTransferNegativeBalance`